src/index.ts

```diff
@@ -1765,8 +1767,4 @@ declare global {
   }
 }
 
-export import JSX = JSXTypes;
```

Comment:
Changing to `import type ...` makes the reexporting of `JSXTypes` fail with TypeScript error `An import alias cannot reference a declaration that was imported using 'import type'.ts(1380)`. So I had to change how the types were exported.